README.md

```diff
@@ -45,198 +29,104 @@ The platform incorporates the following components:
 
-The data analytics and visualization tools and services generate telemetry and log data that can be stored using the platform's time-series database (TSDB) service or by using external tools such as [Elasticsearch](https://www.elastic.co/products/elasticsearch).
-Platform users can easily instrument code and functions to collect various statistics or logs, and explore the collected data in real time.
+You can download additional demos from GitHub &mdash; for example:
 
-The [Grafana](https://grafana.com/grafana) open-source analytics and monitoring framework is natively integrated into the platform, allowing users to create dashboards that provide access to platform NoSQL tables and time-series databases from different dashboard widgets.
-You can also create Grafana dashboards programmatically (for example, from Jupyter Notebook) using wizard scripts.
-For information on how to create Grafana dashboards to monitor and visualize data in the platform, see [Adding a Custom Grafana Dashboard](https://www.iguazio.com/docs/tutorials/latest-release/getting-started/grafana-dashboards/).
+- <a id="xgboost-demo"></a>[**XGBoost classification**](https://github.com/mlrun/demo-xgb-project) &mdash; uses XGBoost to perform binary classification on the popular Iris ML data set, and runs parallel model training with hyperparameters.
+- <a id="image-classification-demo"></a>[**Image classification**](https://github.com/mlrun/demo-image-classification) &mdash; builds and trains an ML model that identifies (recognizes) and classifies (labels) images by using Keras, TensorFlow, and Horovod.
+- <a id="serverless-spark-demo"></a>[**Serverless Spark**](demos/spark/mlrun_sparkk8s.ipynb) &mdash; demonstrates how to run the same Spark job locally and as a distributed MLRun job over Kubernetes.
 
-<a id="end-to-end-use-case-applications"></a>
-## End-to-End Use-Case Applications
+For information on the available demos, contact the Iguazio [support team](mailto:support@iguazio.com).<br>
+For each downloaded demo, start out by reading its **README.md** file.
 
-Iguazio provides full end-to-end use-case applications (demos) that demonstrate how to use the Iguazio Data Science Platform and related tools to address data science requirements for different industries and implementations.
-Some of the demos are pre-deployed with the platform and available in the **demos** tutorial-notebooks directory.<br>
-<a id="mlrun-demos-download"></a>You can get additional demos from the [MLRun demos repository](https://github.com/mlrun/demos) by running the following code.
-> **Note:** Some of the MLRun demos are still works in progress.
+The following example code downloads the [mlrun/demo-xgb-project](https://github.com/mlrun/demo-xgb-project) XGBoost-classification demo:
 
 
 ```python
-# Get MLRun demos
-!chmod +x /User/get-demos.sh
-!/User/get-demos.sh
+# Download the MLRun XGBoost classificaiton demo
+!mlrun project demos/xgboost/ -u git://github.com/mlrun/demo-xgb-project.git
```

Comment:
Regarding using `git clone`, this would also copy Git files (the **.git/** directory and **.gitignore** files), and I'm not sure that's needed — unless the `mlrun` command does the same anyway (I didn't check)?
(The current **get-demos.sh** script does `git clone` to a temporary directory, copies only relevant demo files to the demos directory, and then deletes the temporary directory. But here there was also the issue of only getting some demos from the demos repo.)

Reply:
mlrun project also does git clone, so you'll get the .git directory. If we would like them to be able to update the demos easily then git is a compelling option

Reply:
OK, I replaced the `mlrun` command with `git clone` (commit 9769a86).

Reply:
Looks good. Should it be `cd /User/demos/`?
Do we have the option to do `cd ~` and `mkdir demos`?

Reply:
1. Yes, users can run `cd` and `mkdir` from platform command-line shells such as a Jupyter notebook.
2. The existing get-demos script copies the cloned files to the predefined **/User/demos** directory, which contains the pre-deployed platform demos, so I thought I'd keep the same logic of adding the downloaded demos to the same directory, which makes sense. (There's not need for `mkdir`, because the directory already exists.)
3. I initially thought of setting the path to the demos directory in the `git clone` command instead of doing `cd`, but (a) I wasn't 100% sure whether you provide the path to the local clone directory that you want to create or to its parent directory, and (b) if it's the former (as I think it is), I would need to repeat the name of the demo repo in the command (or give another name), which is a bit more complicated and error prone (especially if we would need to repeat it for each demo). But you can change it, if you want, and you can always test it.

Reply:
Is it our common practice to use /users/demos? I thought we would like to put everything under the user directory by default.

Reply:
As I explained on Slack, it's not `/users/demo`, it's `/User/demos`, which uses the predefined `/User` data mount that maps to `/v3io/users/$V3IO_USERNAME`. When a user creates a Jupyter Notebook service in the platform, we automatically create a **demos** directory with the pre-deployed tutorial demos in the predefined running-user directory in the "users" container — `/User/demos` file-system path — so, it makes sense to add the downloaded demos to this directory, as currently done by the **get-demos.sh** script.